evadb/executor/show_info_executor.py

```diff
@@ -43,5 +45,11 @@ def exec(self, *args, **kwargs):
                 if table.table_type != TableType.SYSTEM_STRUCTURED_DATA:
                     show_entries.append(table.name)
             show_entries = {"name": show_entries}
+        elif self.node.show_type is ShowType.CONFIG:
+            value = self._config.get_value(
```

Comment:
We need to handle cases where the key is not in the config. Return an appropriate error message. Also, add a test case of it. Thanks!

Reply:
Have raised an exception for that for now. Let me know if you think that there is a better way to do it